### ISSUE TITLE
Datetime in data issue

### ISSUE THREAD

**Issue by youpsla** [marcaube/ObHighchartsBundle#19]
Hello,
first of all, I apologize if my question is stupid but I've just spend few hours on my issue and can't find solution.

I'm a beginner in PHP and javascript but I don't think this is the problem here.

My goal is to render a chart similar to the one here http://www.highcharts.com/demo/spline-irregular-time

Here is my code:

In my PHP file:

``` php
    /**
     * @Route("/backoffice/statistics/test", name="test_stats")
     * @Template()
     */
    public function chartAction()
        {
            // Chart

            $series = array(
                array("name" => "Data Serie Name",
                            "data" => '[[1372425426000 ,15 ],[1373426510000 ,20 ],[1373427726000 ,30 ], [1374447726000 ,50 ]]',
            ));
            $ob = new Highchart();
            $ob->chart->renderTo('linechart');  // The #id of the div where to render the chart
            $ob->chart->type('spline');
            $ob->title->text('Chart Title');
            $ob->xAxis->title(array('text'  => "Horizontal axis title"));
            $ob->xAxis->type('datetime');
            $ob->xAxis->dateTimeLabelFormats(array('month'=> '%e. %b', 'year'=> '%b'));
            $ob->yAxis->title(array('text'  => "Vertical axis title"));
            $ob->yAxis->min('0');
            $ob->series($series);
            return $this->render('FloorCRMBundle:Backend:stats.html.twig', array(
                'chart' => $ob, 'title' => 'Statistiques'
            ));
        }

```

And here is my html.twig file:

``` html
{% block javascripts %}
<!-- Load jQuery from Google's CDN if needed -->
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>

<script src="{{ asset('bundles/obhighcharts/js/highcharts/highcharts.js') }}"></script>
<script src="{{ asset('bundles/obhighcharts/js/highcharts/modules/exporting.js') }}"></script>
<script type="text/javascript">
{{ chart(chart) }}
</script>
{% endblock javascripts %}

{% block content %}

<div id="linechart" style="min-width: 400px; height: 400px; margin: 0 auto">

</div>
{% endblock content %}

```

As a result, there is no line on the chart.
I thing there is an issue while transmitting series and especially X datas.

Here is a json of the $ob object. I've this result with the following command:
$json = json_encode( (array)$ob );

{"chart":{"chart":{"renderTo":"linechart","type":"spline"}},"colors":[],"credits":{"credits":{}},"global":{"global":{}},"labels":{"labels":{}},"lang":{"lang":{}},"legend":{"legend":{}},"loading":{"loading":{}},"plotOptions":{"plotOptions":{}},"point":{"point":{}},"series":[[{"name":"Data Serie Name","data":"[[1372425426000 ,15 ],[1373426510000 ,20 ],[1373427726000 ,30 ], [1374447726000 ,50 ]]","pointInterval":3600000,"tickInterval":3600000}]],"subtitle":{"subtitle":{}},"symbols":[],"title":{"title":{"text":"Chart Title"}},"tooltip":{"tooltip":{"formatter":{}}},"xAxis":{"xAxis":{"title":{"text":"Horizontal axis title"},"type":"datetime","dateTimeLabelFormats":{"month":"%e. %b","year":"%b"}}},"yAxis":{"yAxis":{"title":{"text":"Vertical axis title"},"min":"0"}},"exporting":{"exporting":{}},"navigation":{"navigation":{}}} 

If somebody can help, it's very welcome. I thing I'm just close to have it running.

If I put my datas `'[[1372425426000 ,15 ],[1373426510000 ,20 ],[1373427726000 ,30 ], [1374447726000 ,50 ]]'` in the highchrats example at url http://www.highcharts.com/demo/spline-irregular-time , it works !!! Then I suspect data transformation .....

Regards

Alain
